samples/generate-from-code-sample/README.md

```diff
@@ -0,0 +1,42 @@
+## Prerequisites
+You will need:
+1. [JDK 1.8][jdk]
```

Comment:
recommend to add the links both of JDK and Maven

Reply:
ok, I will add the links both of them in the next commit.